lib/measured/unit.rb

```diff
@@ -52,10 +49,6 @@ def inverse_conversion_amount
 
   private
 
-  def case_insensitive(comparison)
-    [comparison].flatten.map(&:downcase)
```

Comment:
This was an unnecessarily high cost to pay for case-insensitive systems. That being said, given the immutability of `Unit` now, we could have cached things.